src/commands/generate/handler.js

```diff
@@ -13,6 +13,10 @@ internals.generate = async (url = '', options = {}) => {
   });
   const page = await browser.newPage();
 
+  if (options.viewportSize) {
+    await page.setViewport(options.viewportSize);
```

Comment:
I'm thinking a `try/catch` statement is warranted here. It would be too easy to mistype `200x350` and cause an error here.